draft-ietf-trans-gossip.md

```diff
@@ -401,30 +409,43 @@ with the following content:
     representation of the binary SCT data as defined in
     {{RFC-6962-BIS-09}} Section 3.3.
 
-The 'x509\_chain' element MUST contain at least the leaf certificate
-and SHOULD contain the full chain to a root accepted by all of the
-logs in the set of logs issuing all the SCTs in the 'sct\_data'
-element.
+The 'x509\_chain' element MUST contain the full chain from the leaf certificate 
+to a trust anchor (either pre-loaded or locally added) accepted by all of the 
+logs in the set of logs issuing all the SCTs in the 'sct\_data' element.
+
+\[ 
+  TODO: I'm sorry, I know we discussed this before, but I again take concern with
+  "a trust anchor accepted by all of the logs in the set of logs issuing all 
+    the SCTs in the 'sct\_data' element."
+  The client doesn't know if the log accepts the root it chained to. All it knows is
+  "This is an SCT from a log I trust." and "This is the cert chain I built."
+
+  What was the rational for keeping that phrase in? Shouldn't it just be: 
+  The 'x509\_chain' element MUST contain the full chain from the leaf certificate 
+  to a trust anchor.
+\]
+
```

Comment:
> The rationale is that this protects exposure of locally added logs and/or CA:s. Can't remember if both but I'm sure dkg knows! Is he here? Should I move this discussion back to the draft text?

I'm not sure I grok this fully. We can talk about it on the call.

> Also, the client does know that a log which has issued an SCT for a cert accepts the root it's chained to -- if not, the log wouldn't have accepted it and thus not issued an SCT.

Not necessarily. a) The log may have accepted the cert when it chained up to a _different_ root than the one the client built a path to.  b) The log may have trusted the root at the point in time the cert was entered, but not longer does.  Again, call.